index.rst

```diff
@@ -4,20 +4,715 @@
+If necessary, a workflow system might be interposed.
+
+
+.. _level-2-domain:
+
+Level 2 Domain
+==============
+
+This domain is responsible for all longer-period data processing operations, including the largest and most complex payloads supported by the DMS: the annual Data Release Production (DRP) and periodic Calibration Products Productions (CPPs).
+Note that CPPs will execute even while the annual DRP is executing, hence the need for a separate service.
+The Level 2 Quality Control Service monitors the science data products, notifying operators if any anomalies are found.
+
+The services in this domain need to run efficiently and reliably over long periods of time, spanning weeks or months.
+They need to execute millions or billions of tasks when their input data is available while tracking the status of each and preserving its output.
+They are designed to execute autonomously with human oversight, monitoring, and control primarily at the highest level, although provisions are made for manual intervention if absolutely necessary.
+
+This domain does not have direct users (besides the operators of its services); the services within it obtain inputs from the Data Backbone and place their outputs into the Data Backbone.
+
+
+.. _level-2-services:
+
+Service Descriptions
+--------------------
+
+.. _cpp-execution:
+
+Calibration Products Production Execution service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This executes various CPP science payloads at various intervals to generate Master Calibration Images and populate the Calibration Database with information derived from analysis of raw calibration images from the Data Backbone and information in the Transformed EFD.
+This includes the computation of crosstalk correction matrices.
+Although not a calibration product, the templates used by Alert Production are also generated by this service, based on raw science images from the Data Backbone.
+Additional information such as external catalogs may also be taken from the Data Backbone.
+Computations occur using the Flexibly-Provisioned Compute Resources.
+The intervals at which this service executes will depend on the stability of Observatory systems, but are expected to include at least monthly and annual executions.
+The annual execution is a prerequisite for the subsequent execution of the Data Release Production.
+The service involves human scientist/operator input to determine initial configurations of the payload, to monitor and analyze the results, and possibly to provide additional configuration information during execution.
+
+Requirements satisfied: [...]
+
+.. _drp-execution:
+
+Data Release Production Execution service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This executes the DRP science payload annually to generate all Level 2 data products after the annual CPP is executed.
+A small-scale (about 10% of the sky) mini-production is executed first to ensure readiness, followed by the full production.
```

Comment:
I would expect that most small-scale testing would be done before the mini-production using the developer and integration environments, and that any debugging needed during or after the mini-production would generally use those same environments.  The intent of the mini-production is to be a final validation, not to be the first time the data is ever processed.

Mini-production (and pre-release DRP) data products can be made available to external scientists, but they should be individuals considered part of the Science Validation team who would be granted access to internal systems (in particular the Science Validation instance of the LSST Science Platform).  Making pre-release data products available through the DACs does not sound like a good idea, except possibly as an exceptional case during Commissioning.